awx/api/views/__init__.py

```diff
@@ -209,6 +209,26 @@ def api_exception_handler(exc, context):
     return exception_handler(exc, context)
 
 
+class LabelList(DeleteLastUnattachLabelMixin, SubListCreateAttachDetachAPIView):
```

Comment:
class name is a little confusing, would be better to say `LabelSubList` or something /nit